pages/verification/index.tsx

```diff
@@ -1,19 +1,19 @@
-import { BaseContainer } from "@components/Page/BaseContainer";
-import { Helmet } from "react-helmet";
-import React, { useState } from "react";
+import Cookies from "js-cookie";
 import LayoutContainer from "@components/LayoutContainer";
 import api from "@shared/fetcher";
 import apiAnonymous from "@shared/fetcherAnonymous";
-import { useCheckExchangeTokenMutation } from "@services/auth/authHooks";
-import { useRouter } from "next/router";
+import getConfig from "next/config";
+import React, { useState } from "react";
+import { BaseContainer } from "@components/Page/BaseContainer";
 import { GetServerSidePropsContext } from "next";
-import Cookies from "js-cookie";
+import { Helmet } from "react-helmet";
 import { ITokenEnum, MessageEnum, UserEnum } from "@shared/enum";
-import { useInitChatAnonymousMutation } from "@services/initChatAnonymous/initChatAnonymousHooks";
+import { LoaderWrapper } from "@components/LoaderWrapper";
 import { toast } from 'react-toastify'
-import getConfig from "next/config";
+import { useCheckExchangeTokenMutation } from "@services/auth/authHooks";
 import { useGenerateAnonUserInfoMutation } from "@services/generateAnonUserInfo/generateAnonUserInfoHooks";
-import { LoaderWrapper } from "@components/LoaderWrapper";
+import { useInitChatAnonymousMutation } from "@services/initChatAnonymous/initChatAnonymousHooks";
+import { useRouter } from "next/router";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been reordered. While this doesn't affect the functionality of the code, it's generally a good practice to maintain a consistent order for import statements. This helps in improving readability and maintainability of the code. However, since there is no standard rule for ordering imports in JavaScript or TypeScript, this change is acceptable as long as it follows the project's conventions.

```diff
- import { BaseContainer } from "@components/Page/BaseContainer";
- import { Helmet } from "react-helmet";
- import React, { useState } from "react";
- import LayoutContainer from "@components/LayoutContainer";
- import api from "@shared/fetcher";
- import apiAnonymous from "@shared/fetcherAnonymous";
- import { useCheckExchangeTokenMutation } from "@services/auth/authHooks";
- import { useRouter } from "next/router";
- import { GetServerSidePropsContext } from "next";
- import Cookies from "js-cookie";
- import { ITokenEnum, MessageEnum, UserEnum } from "@shared/enum";
- import { useInitChatAnonymousMutation } from "@services/initChatAnonymous/initChatAnonymousHooks";
- import { toast } from 'react-toastify'
- import getConfig from "next/config";
- import { useGenerateAnonUserInfoMutation } from "@services/generateAnonUserInfo/generateAnonUserInfoHooks";
- import { LoaderWrapper } from "@components/LoaderWrapper";
+ import Cookies from "js-cookie";
+ import LayoutContainer from "@components/LayoutContainer";
+ import api from "@shared/fetcher";
+ import apiAnonymous from "@shared/fetcherAnonymous";
+ import getConfig from "next/config";
+ import React, { useState } from "react";
+ import { BaseContainer } from "@components/Page/BaseContainer";
+ import { GetServerSidePropsContext } from "next";
+ import { Helmet } from "react-helmet";
+ import { ITokenEnum, MessageEnum, UserEnum } from "@shared/enum";
+ import { LoaderWrapper } from "@components/LoaderWrapper";
+ import { toast } from 'react-toastify'
+ import { useCheckExchangeTokenMutation } from "@services/auth/authHooks";
+ import { useGenerateAnonUserInfoMutation } from "@services/generateAnonUserInfo/generateAnonUserInfoHooks";
+ import { useInitChatAnonymousMutation } from "@services/initChatAnonymous/initChatAnonymousHooks";
+ import { useRouter } from "next/router";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->